coherence/coherence.go

```diff
@@ -0,0 +1,128 @@
+package coherence
+
+/*
+ * Copyright © 2023, Oracle and/or its affiliates.
+ */
+import (
+	"context"
+	coh "github.com/oracle/coherence-go-client/coherence"
+	"time"
+)
+
+const (
+	defaultScopeName = "fiber-session-store"
+	defaultTimeout   = time.Duration(30) * time.Millisecond
+)
+
+// Storage represents an implementation of Coherence storage provider.
+type Storage struct {
+	session    *coh.Session
+	namedCache coh.NamedCache[string, []byte]
+	ctx        context.Context
+}
+
+// Config defines configuration options for Coherence connection.
+type Config struct {
+	// Address to connect to, defaults to "localhost:1408"
+	Address string
+
+	// UseSSL specified if to use SSL or plain text, defaults to false
+	UseSSL bool
```

Comment:
This should be a tls Config, like the other drivers. Else it won't work for private/self-signed/corporate environments

Reply:
Currently we only support setting these via environment variables as described here - https://pkg.go.dev/github.com/oracle/coherence-go-client/coherence#NewSession
I'm in the process of changing this, but may be able to do a workaround in the meantime.

Reply:
That's fine, else it can be added as a feature after initial release.